airbyte-webapp/src/components/TextWithHTML/TextWithHTML.tsx

```diff
@@ -1,30 +1,24 @@
 import React from "react";
-import sanitizeHtml from "sanitize-html";
+import DOMPurify from "dompurify";
 
 type IProps = {
   text?: string;
 };
 
-const allowedAttributes = {
-  ...sanitizeHtml.defaults.allowedAttributes,
-  a: [...sanitizeHtml.defaults.allowedAttributes["a"], "rel"],
-};
-
 const TextWithHTML: React.FC<IProps> = ({ text }) => {
   if (!text) {
     return null;
   }
 
-  const sanitizedHtmlText = sanitizeHtml(text, {
-    allowedAttributes,
-    transformTags: {
-      a: sanitizeHtml.simpleTransform("a", {
-        target: "_blank",
-        rel: "noopener noreferrer",
-      }),
-    },
+  DOMPurify.addHook("afterSanitizeAttributes", function (node) {
+    if ("rel" in node) {
+      node.setAttribute("target", "_blank");
+      node.setAttribute("rel", "noopener noreferrer");
+    }
   });
 
+  var sanitizedHtmlText = DOMPurify.sanitize(text, { ALLOWED_TAGS: ["a"], ALLOWED_ATTR: ["rel"] });
```

Comment:
We're using modern TS syntax across the codebase. Thus this `var` should be a `const` instead. We also have eslint setup to check for this. This should have shown a warning on your console (and an overlay in the UI) when you tried to run this. In case that didn't work in your setup you can always run `npm run lint` in the airbyte-webapp folder to run eslint manually.